src/renderer/runner.ts

```diff
@@ -182,7 +194,10 @@ export class Runner {
       return RunResult.INVALID;
     }
 
-    const isReady = getIsDownloaded(version, localPath);
+    const isReady =
+      state === InstallState.installed ||
+      state === InstallState.downloaded ||
```

Comment:
question: It's been a while since I've taken a look at `fiddle-core` code. What is the difference between `InstallState.installed` and `InstallState.downloaded` if they are both considered ready to run here?

Reply:
`downloaded` is the state when the binary is downloaded and kept as a `zip` file whereas the `installed` is when the downloaded binary is currently unzipped for the fiddle to run (`fiddle-core` doesn't unzip all downloaded versions to improve disk space usage)
In our use case - We want both the states to be marked `ready` since `fiddle-core.spawn` by default unzips the version if it's not installed.